app/controllers/readings_controller.rb

```diff
@@ -12,10 +12,18 @@ def load_user
   def index
     @readings = @user.readings
     @page_subtitle = ts("History")
+    @kudos_list = []
     if params[:show] == 'to-read'
       @readings = @readings.where(:toread => true)
       @page_subtitle = ts("Saved For Later")
     end
+    if params[:show] == 'kudos-history'
+      # collext a list of pseuds the user may have left kudos under
+      pseuds=Pseud.where(user_id: @user.id ).value_of(:id)
```

Comment:
Surrounding space missing for operator '='.<br>Space inside parentheses detected.